core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -37,6 +42,56 @@
  */
 public class LoadTableResponse implements RESTResponse {
 
+  private static final String METADATA_LOCATION = "metadata-location";
+  private static final String METADATA = "metadata";
+  private static final String CONFIG = "config";
+
+  public static void toJson(LoadTableResponse loadTable, JsonGenerator generator) throws IOException {
+    generator.writeStartObject();
+
+    generator.writeStringField(METADATA_LOCATION, loadTable.metadataLocation);
+
+    generator.writeFieldName(METADATA);
+    TableMetadataParser.toJson(loadTable.metadata, generator);
+
+    generator.writeObjectFieldStart(CONFIG);
+    for (Map.Entry<String, String> keyValue : loadTable.config.entrySet()) {
+      generator.writeStringField(keyValue.getKey(), keyValue.getValue());
+    }
+    generator.writeEndObject();
+
+    generator.writeEndObject();
+  }
+
+  public static LoadTableResponse fromJson(JsonNode node) {
+    Preconditions.checkArgument(node.isObject(),
+            "Cannot parse table response from a non-object: %s", node);
+
+    String metadataLocation = null;
+    JsonNode  metadataLocationNode = node.get(METADATA_LOCATION);
+    if(metadataLocationNode != null) {
+      metadataLocation = metadataLocationNode.asText();
+    }
```

Comment:
You can use `String metadataLocation = JsonUtil.getStringOrNull(METADATA_LOCATION, node)` for convenience here.

Reply:
Ooh, that's nice. Thanks!